internal/tiltfile/skylarkTypes.go

```diff
@@ -76,11 +76,12 @@ type mount struct {
 }
 
 type dockerImage struct {
-	fileName   string
-	fileTag    reference.Named
-	mounts     []mount
-	steps      []model.Step
-	entrypoint string
+	fileName     string
+	fileTag      reference.Named
+	mounts       []mount
+	steps        []model.Step
+	entrypoint   string
+	pathMatchers []model.PathMatcher
```

Comment:
might be work calling these filters to indicate that they're the files we're ignoring